src/react/actions/__tests__/s3object.test.jsx

```diff
@@ -264,6 +286,134 @@ describe('s3object actions', () => {
                 dispatchAction.NETWORK_END_ACTION,
             ],
         },
+        {
+            it: 'getObjectMetadata: should return expected actions -> test without prefix parameter',
+            fn: actions.getObjectMetadata(BUCKET_NAME, '', OBJECT_KEY),
+            storeState: initState,
+            expectedActions: [
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.GET_OBJECT_METADATA_SUCCESS_ACTION(BUCKET_NAME, '', OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
```

Comment:
Can we provide a non-empty tags array? (type `TagSet`)

Reply:
done